protocol/avdl/keybase1/kbfs_common.avdl

```diff
@@ -101,4 +101,10 @@ protocol kbfsCommon {
     array<string> syncingPaths;
     union { null, Time} endEstimate;
   }
+
+  record FSOverallSyncStatus {
+    int64 syncingBytes;
```

Comment:
These two fields could use either new names, or some comments. I can't figure out which one is supposed to be the total number of bytes we know about that needed syncing, and the amount of those that we've synced so far.  (Or is it the amount we still have left to sync?)

In `PrefetchProgress`, we use:

```
    int64 bytesTotal;
    int64 bytesFetched;
```

which I think it more clear.

Reply:
ok, will rename to that if we don't switch to the other struct entirely.